src/components/AccordionList.jsx

```diff
@@ -11,11 +11,12 @@ export default function SimpleAccordion() {
     <div data-test="component-accordionList">
       <Accordion>
         <AccordionSummary
-          expandIcon={<ExpandMoreIcon />}
+          expandIcon={<ExpandMoreIcon sx={{ color: "white" }} />}
```

Comment:
❓ Is there a way to access `white` from the theme? Maybe `color: palette.primary.white` or whatever is appropriate. That goes for everything else as well.

Reply:
Addressed:  ff26e8bef482688b80219260dc39ed75b9aae61b